tensorboard/plugins/image/summary_v2.py

```diff
@@ -27,6 +27,41 @@
 def image(name, data, step=None, max_outputs=3, description=None):
     """Write an image summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`.
+
+    Logs a set of images to the current log directory for later analysis
+    in TensorBoard's 'Images' dashboard. Like `tf.summary.scalar` points,
+    each set of images is associated with a `step` and a `name`.  All the
+    image sets with the same `name` constitute a time series of image sets.
+
+    This example writes 2 random grayscale images:
+
+    ```python
+    test_summary_writer = tf.summary.create_file_writer('test/logs')
```

Comment:
Optional but I might use a shorter name like `w` here to be a bit more colloquial/brief for an example.

Reply:
Done, and will do in the .histogram PR as well.